### ISSUE TITLE
Can't create new file unless folder/file is selected

### ISSUE THREAD

**Issue by OscarGodson** [OscarGodson/New-File#2]
A feature I really want. Not sure how to implement it yet.

**Comment by garyking** [comment]
Yes indeed, this would be great! It means that at the moment, I have to move up a folder just to create a new file in my current folder.

**Comment by VictorChen** [comment]
+1

**Comment by RowanKaag** [comment]
+1

**Comment by quantumgolem** [comment]
I don't think you can implement it into the context menu with Automator because of the way that macOS works. However, I think it's possible with a shortcut in the Finder.
